src/components/NavBar.jsx

```diff
@@ -6,11 +6,11 @@ export const NavBar = (props) => {
     return ( 
         <ul {...props}>
             <li key='home'>
-                <Link className="text-sm text-gray-800 hover:text-gray-950" to="/">Home</Link>
+                <Link className=" text-[#eb1b24] hover:text-gray-950" to="/">Home</Link>
             </li>
             {topics.map((topic, index) => (
```

Comment:
If we can use any other unique value than the index, it's best practise to. For example, the key could be the slug :)